[2024-05-31T08:38:12.794+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-31T08:38:12.814+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.load_results manual__2024-05-31T08:38:04.847215+00:00 [queued]>
[2024-05-31T08:38:12.823+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.load_results manual__2024-05-31T08:38:04.847215+00:00 [queued]>
[2024-05-31T08:38:12.825+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-31T08:38:12.835+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_results> on 2024-05-31 08:38:04.847215+00:00
[2024-05-31T08:38:12.843+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=889) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-31T08:38:12.845+0000] {standard_task_runner.py:63} INFO - Started process 890 to run task
[2024-05-31T08:38:12.845+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'load_results', 'manual__2024-05-31T08:38:04.847215+00:00', '--job-id', '582', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmp2b77qmmk']
[2024-05-31T08:38:12.847+0000] {standard_task_runner.py:91} INFO - Job 582: Subtask load_results
[2024-05-31T08:38:12.898+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.load_results manual__2024-05-31T08:38:04.847215+00:00 [running]> on host 29c09f03cc50
[2024-05-31T08:38:12.980+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='load_results' AIRFLOW_CTX_EXECUTION_DATE='2024-05-31T08:38:04.847215+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-31T08:38:04.847215+00:00'
[2024-05-31T08:38:12.982+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-31T08:38:13.423+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-31T08:38:13.424+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-31T08:38:13.435+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=load_results, execution_date=20240531T083804, start_date=20240531T083812, end_date=20240531T083813
[2024-05-31T08:38:13.461+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-31T08:38:13.491+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-31T08:38:13.495+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
